netci.groovy

```diff
@@ -103,7 +103,7 @@ set TMP=%TEMP%
 
       def triggerPhraseOnly = configuration == 'release'   
       def triggerPhraseExtra = ""
-                Utilities.setMachineAffinity(myJob, 'Windows_NT', 'latest-or-auto')
+                Utilities.setMachineAffinity(myJob, 'Windows_NT', 'latest-or-auto-update3')
```

Comment:
This has the words "latest" and "update3" which seem to be in conflict.  Why not just "update3"?